docs/dev/agent_build.md

```diff
@@ -45,6 +45,8 @@ This is the complete list of the available components:
 Please note you might need to provide some extra dependencies in your dev
 environment to build certain bits (see [development environment][dev-env]).
 
+Also note that the trace agent needs to be built and run separately: see [here](./trace-agent/READE.md).
```

Comment:
```suggestion
Also note that the trace agent needs to be built and run separately: see [here](../trace-agent/README.md).
```

Reply:
I'm not sure but don't you need `..` before to refer to `docs/trace-agent` folder as you are in the `docs/dev` folder ?

Reply:
Thanks, fixed 😳